components/form/__tests__/__snapshots__/demo-extend.test.ts.snap

```diff
@@ -7532,6 +7588,7 @@ exports[`renders components/form/demo/label-debug.tsx extend context correctly 1
             class="ant-form-item-control-input-content"
           >
             <input
+              aria-label="[object Object]"
```

Comment:
This should be a wrong aria-label.

Reply:
sorry It was my negligence.The problem 'aria-label="[object Object]' arises because the label property of FormItem is a component.if the prop label of FormItem is component , then the aria-label of its child components should be what?

Reply:
@ yoyo837 @afc163  @zombieJ Sorry,it was my negligence.The problem 'aria-label="[object Object]' arises because the label property of FormItem is a component.if the prop label of FormItem is component , then the aria-label of its child components should be what?
![image](https://github.com/ant-design/ant-design/assets/62460957/def81a11-6f7f-45cf-928e-92268433216c)

Reply:
@yoyo837 @afc163 @zombieJ Sorry,it was my negligence.The problem 'aria-label="[object Object]' arises because the label property of FormItem is a component.if the prop label of FormItem is component , then the aria-label of its child components should be what?
![1696648319017](https://github.com/ant-design/ant-design/assets/62460957/96528d20-79f2-42e1-a7bd-c22e72972221)

Reply:
`renderToString`?

Reply:
@yoyo837  Extracting text from a component? In the example,regard 'longtextlongtext...' as input aria-label? What if label is an empty tag for example '<div></div>'?

Reply:
@yoyo837 Extracting text from a component? In the example,regard 'longtextlongtext...' as input aria-label? What if label is an empty tag for example `<div></div>`?

Reply:
Maybe other members have good suggestions... 🤔

Reply:
I have an idea. if label is a component, the children property of props is searched recursively until children are of type String.If not , the default 'Search ' is used as the value of the aria-label property .

Reply:
Not sure I understand completely, but how about `domElement.innerText`? That should handle both string and component children.

Reply:
I know what you mean.But label is React.ReactNode not HTMLElement.What I want to do is as follows.
```
function getTextFromElement(element) {
  let text = '';

  React.Children.map(element.props.children, child => {
    if (React.isValidElement(child)) {
      text += getTextFromElement(child);
    } else if (typeof child === 'string') {
      text += child;
    }
  });

  return text;
}

const A = (
  <div>
    Hello <span>world</span>!
  </div>
);

const text = getTextFromElement(A);
console.log(text); // Output: "Hello world!"
```

Reply:
@yoyo837 @afc163 @zombieJ I fix wrong aria-label. If you have time, please review Pr.